### PR TITLE
Update Swift SDK meeting notes URL

### PR DESCRIPTION
**Pull request by nachoBonafonte**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -140,7 +140,7 @@ PHP: SDK|Every Wednesday at 10:30 PT|[Google Doc](https://docs.google.com/docume
 Python: SDK|Every Thursday at 09:00 PT|[Google Doc](https://docs.google.com/document/d/1CIMGoIOZ-c3-igzbd6_Pnxx1SjAkjwqoYSUWxPY8XIs/edit)|[Slack](https://cloud-native.slack.com/archives/C01PD4HUVBL)|
 Ruby: SDK|Every Tuesday at 09:00 PT|[Google Doc](https://docs.google.com/document/d/1D15bO8o340sQm2CVZiukEJuCO_XMMHKPuTznoEhyFqE/edit)|[Slack](https://cloud-native.slack.com/archives/C01NWKKMKMY) and [GitHub Discussions](https://github.com/open-telemetry/opentelemetry-ruby/discussions)|
 Rust: SDK|Every other week on Tuesday alternating between 07:00 and 15:00 PT|[Google Doc](https://docs.google.com/document/d/1tGKuCsSnyT2McDncVJrMgg74_z8V06riWZa0Sr79I_4)|[Gitter](https://gitter.im/open-telemetry/opentelemetry-rust)|
-Swift: SDK|Every Thursday at 09:00 PT|[Google Doc](https://docs.google.com/document/d/1ShEFMywIV4LJcDYCNy41zkq8RR1sMq-tSvMBHngHcuk/edit?usp=sharing)|[Slack](https://cloud-native.slack.com/archives/C01NCHR19SB)||
+Swift: SDK|Every Thursday at 09:00 PT|[Google Doc](https://docs.google.com/document/d/17KAyOXt-0Ti3PVB5_tLJi3lFeFNfp_98Ji3NocBqLQI/edit)|[Slack](https://cloud-native.slack.com/archives/C01NCHR19SB)||
```
Comment (arminru) [inline README.md:143]: This document is not publicly accessible. Please make sure it's world-readable (maybe allow comment access, most SIGs even have them publicly editable although that's probably not ideal).

Please also make sure it's owned by the OTel GC account - see https://github.com/open-telemetry/community/issues/1147.

Comment (dyladan) [reply README.md:143]: @nachoBonafonte any update? The document is still private.

Comment (trask) [reply README.md:143]: ping @open-telemetry/swift-maintainers

**Comment by nachoBonafonte** [comment]
Sorry for the delay, I totally forgot about it. I have tried assigning ownership as explained in #1147  but was not able to do it. Transfer ownership does not appear as an option after adding cncf as editor. At the end I have created a public accessible file with my personal ownership.

**Comment by arminru** [comment]
@nachoBonafonte Now that you have a doc owned from your personal account, could you try following the steps @dyladan laid out in https://github.com/open-telemetry/community/issues/1147#issuecomment-1234759420? I think now passing ownership should work.

**Comment by nachoBonafonte** [comment]
I created the ownership transfer now, waiting for the acceptance

**Comment by trask** [comment]
> I created the ownership transfer now, waiting for the acceptance

accepted, thanks!

**Review by trask — APPROVED** [review]

**Comment by trask** [comment]
@open-telemetry/governance-committee can we get one more approval on this? thx!

**Comment by trask** [comment]
@open-telemetry/governance-committee we still need another approval on this

@nachoBonafonte can you resolve the merge conflict? (sorry we didn't get this merged before conflicts arose)

**Review by yurishkuro — APPROVED** [review]

**Pull request merged**